README.md

```diff
@@ -1,16 +1,10 @@
-### Hi there 👋
+# About me ✨✨✨
 
-<!--
-**Alisa411/Alisa411** is a ✨ _special_ ✨ repository because its `README.md` (this file) appears on your GitHub profile.
-
-Here are some ideas to get you started:
-
-- 🔭 I’m currently working on ...
-- 🌱 I’m currently learning ...
-- 👯 I’m looking to collaborate on ...
-- 🤔 I’m looking for help with ...
-- 💬 Ask me about ...
-- 📫 How to reach me: ...
-- 😄 Pronouns: ...
-- ⚡ Fun fact: ...
--->
+- 🔭 I’m currently working on ***X-linked intellectual disability*** disease modeling based on IPSc
+- 🌱 I’m currently learning **bioinformatics**
+- 👯 I’m looking to collaborate with other students interested in bioinformatics
+- 🤔 I’m looking for help with python and bash
+- 💬 Ask me about 
+> what I like the most. And I will answer - sleeeeep
```

Comment:
He-he, the GitHub have a nice one presaved template, yeah. BTW i liked that you added some Markdown features like bold / italic text and quotes.